### DISCUSSION TITLE
Get help with compiling some projects!

### DISCUSSION THREAD

**Discussion by sorydima** [codemagic-ci-cd/codemagic-docs#2413] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Just visit 
https://github.com/sorydima/REChain-.git
and
https://github.com/sorydima/Katya-.git

I could not compile Katya for both ios and macos , and REChain for macos. So what should i do? 

### Expected results

Compiling Katya for ios and macos 
Compiling REChain for macos

### Actual results

errors with framewroks such as RTC Frame cryptor (.m)
errors with flutter versions etc.
commits are welcome!!!

**Comment by dtrdic** [comment, marked as answer]
Hi @sorydima sounds like you're dealing with Flutter development. For Flutter development-specific queries, it might be more beneficial to seek help from the broader [Flutter community](https://github.com/fluttercommunity)forums. They are likely to provide more better assistance for Flutter-related development issues. If you have any Codemagic-specific questions, feel free to ask
